rules/windows/defense_evasion_stop_process_service_threshold.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/12/03"
+maturity = "production"
+updated_date = "2020/12/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule identifies high number of evasion related process executions (stop, delete or suspend a process or service)
+from the same host and within a short time period. This may indicate defense evasion attempt.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "High Number of Process and/or Service Termination"
```

Comment:
the events won't all be terminations, no? more like Modifications?

Reply:
yep all process termination or stopping services (backup, security etc.), some ransomware launch a series of taskkill and sc stop etc.

Reply:
sounds good, i'll go back and change the description then